projects/addon-editor/components/toolbar-tools/code/code.component.ts

```diff
@@ -38,3 +43,10 @@ export class CodeComponent {
         }
     }
 }
+
+@Component({
+    selector: 'tui-code',
```

Comment:
Two components cannot match the same selector. I think we can just rename the class.

Reply:
fixed